packages/transaction-controller/src/types.ts

```diff
@@ -54,6 +60,24 @@ type TransactionMetaBase = {
    */
   deviceConfirmedOn?: WalletDevice;
 
+  /**
+   * [swaps-only]
```

Comment:
Is this needed? Lots of the properties here are specific to certain use cases but is it not clear from the descriptions that it's for swaps transactions?

Reply:
Removed all in https://github.com/MetaMask/core/pull/1877/commits/54ec5d3b7a2cef7e859ef1335c34fd8f64cd2e3d